src/main/java/com/neonlab/product/dtos/ProductDto.java

```diff
@@ -1,11 +1,15 @@
 package com.neonlab.product.dtos;
+import com.neonlab.common.dto.DocumentDto;
 import com.neonlab.product.enums.Units;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+
 import java.math.BigDecimal;
 
 
+@EqualsAndHashCode(callSuper = true)
 @Data
-public class ProductDto {
+public class ProductDto extends DocumentDto {
```

Comment:
Why?

cc- @sjha24

Reply:
Only add the list<String> documentId that is sufficient for frontend to call subsequent api to fetch the doucments

cc- @sjha24